MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -433,10 +501,25 @@ - (void)acquireTokenInteractive:(id)sender
                 [self updateResultViewError:error];
             }
             
+            [_webView loadHTMLString:@"<html><head></head><body>done!</body></html>" baseURL:nil];
```

Comment:
why do we need a "done!" string :)?

Reply:
Pure legacy, I am fine removing this in this new world.